trunk/source/TraceAbstraction/src/de/uni_freiburg/informatik/ultimate/plugins/generator/traceabstraction/tracehandling/StrategyFactory.java

```diff
@@ -235,19 +255,29 @@ public class StrategyModuleFactory {
 
 		private final TaskIdentifier mTaskIdentifier;
 		private final IUltimateServiceProvider mServices;
-		private final IRun<L, ?> mCounterexample;
+		private final Counterexample<L> mCounterexample;
 		private final IPredicateUnifier mPredicateUnifier;
 		private final IPredicate mPrecondition;
 		private final IPredicate mPostcondition;
 		private final IAutomaton<L, IPredicate> mAbstraction;
 		private final IEmptyStackStateFactory<IPredicate> mEmptyStackFactory;
 
+		/**
+		 * @param counterexample
+		 *            A trace that will be checked for feasibility and for which, if it is infeasible, a refinement
+		 *            result will be constructed.
+		 * @param controlConfigurationSequence
+		 *            Optionally, the sequence of control configurations visited by the trace in the program that is
+		 *            being verified. Used to judge the quality of proofs ("perfect") and for assert order modulation.
+		 * @param abstraction
+		 *            TODO ???
```

Comment:
Because I wanted to document this parameter. I did so for the `constructStrategy` methods above, but forgot it here. Fixed now, and I also updated the rest of the documentation block for the removal of the `controlConfigurationSequence` parameter.